src/bp-templates/bp-nouveau/buddypress/common/js-templates/activity/parts/bp-activity-link-preview.php

```diff
@@ -9,26 +9,36 @@
  */
 
 ?>
-<script type="text/html" id="tmpl-activity-link-preview">
+<script type="text/html" id="tmpl-activity-link-preview">	
 	<# if ( data.link_scrapping ) { #>
 	<# if ( data.link_loading ) { #>
 	<span class="activity-url-scrapper-loading activity-ajax-loader"><i class="bb-icon-l bb-icon-spinner animate-spin"></i><?php esc_html_e( 'Loading preview...', 'buddyboss' ) ?></span>
 	<# } #>
 	<# if ( data.link_success || data.link_error ) { #>
 	<a title="<?php esc_html_e( 'Cancel Preview', 'buddyboss' ); ?>" href="#" id="activity-close-link-suggestion"><?php esc_html_e( 'Remove Preview', 'buddyboss' ); ?></i></a>
-	<div class="activity-link-preview-container">
+	<div class="activity-link-preview-container">		
 
-		<# if ( data.link_success && ! data.link_error ) { #>
-		<p class="activity-link-preview-title">{{{data.link_title}}}</p>
-		<# } #>
-
-		<# if ( data.link_images && data.link_images.length && data.link_success && ! data.link_error ) { #>
+		<# if ( data.link_images && data.link_images.length && data.link_success && ! data.link_error && '' !== data.link_image_index ) { #>
 		<div id="activity-url-scrapper-img-holder">
 			<div class="activity-link-preview-image">
-				<img src="{{{data.link_images[data.link_image_index]}}}"/>
-				<a title="Cancel Preview Image" href="#" id="activity-link-preview-close-image">
+				<div class="activity-link-preview-image-cover">
+					<img src="{{{data.link_images[data.link_image_index]}}}"/>
+				</div>
+				<!-- <a title="Cancel Preview Image" href="#" id="activity-link-preview-close-image">
 					<i class="bb-icon-l bb-icon-times"></i>
-				</a>
+				</a>-->
+				<div class="activity-link-preview-icons">
+					<#
+					if ( data.link_images.length > 1 ) { #>
+						<a data-bp-tooltip-pos="up" data-bp-tooltip="Change image" href="#" class="icon-exchange toolbar-button bp-tooltip" id="icon-exchange"><i class="bb-icon-l bb-icon-exchange"></i></a>
+					<# } #>
+					<# if ( data.link_images.length ) { #>
+						<a data-bp-tooltip-pos="up" data-bp-tooltip="Remove image" href="#" class="icon-image-slash toolbar-button bp-tooltip" id="activity-link-preview-remove-image"><i class="bb-icon-l bb-icon-image-slash"></i></a>
```

Comment:
```suggestion
						<a data-bp-tooltip-pos="up" data-bp-tooltip="Remove image" href="#" class="icon-image-slash toolbar-button bp-tooltip" id="activity-link-preview-remove-image"><i class="bb-icon-l bb-icon-image-slash"></i></a>
```
Need to add a translation function in **Remove image**